cal/cal/static/css/calendaranalytics.scss

```diff
@@ -20,6 +20,10 @@ button {
   margin: 2px;
 }
 
+a[ng-click]{
```

Comment:
This was a hacky way to prevent link clicking to go up to the top of the page. Your solution also combatted that so I amended it, thanks!